web/person-roles/app/pages/officer-change/[businessId].vue

```diff
@@ -19,10 +19,23 @@ useHead({
 
 definePageMeta({
   layout: 'filing',
-  middleware: () => {
-    // redirect to reg home with return url if user unauthenticated
+  middleware: async () => {
+    // mock user if in CI environment // TODO: figure out real logins for e2e in CI
     const { $connectAuth, $config } = useNuxtApp()
-    if (!$connectAuth.authenticated) {
+    if ($config.public.ci) {
```

Comment:
instead of using the 'ci' variable you may want to do what I did in this pr: https://github.com/bcgov-registries/beneficial-ownership/pull/379/files?file-filters%5B%5D=.ts&show-deleted-files=true&show-viewed-files=true#diff-208aa7b01c447743371525a054e890d6e02ba6fe5cca212a885a78675737e989

That way its checking for playwright running (nice for local runs as well)